packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/SVGPanZoom/SVGPanZoom.js

```diff
@@ -104,28 +115,29 @@ function SVGPanZoom({
         return
       }
       case 'zoomto': {
-        setZoom(1)
         setViewBox({
           x: 0,
           y: 0,
-          width: naturalWidth,
-          height: naturalHeight
+          height: naturalHeight,
+          width: naturalWidth
         })
+        setZoom(1)
+        return
       }
     }
   }
 
   const { x, y, width, height } = viewBox
-  const scale = imageScale(img)
+  const scale = imageScale(imgRef, naturalWidth)
 
   return (
-    <div style={{ width: '100%' }}>
+    <FullWidthDiv>
```

Comment:
Out of curiosity, does moving the `width` property to styled-components from the style object have an impact on the UI or is it a dx preference? I ask because styled-components has been brought up in FE discussion as a tech debt, and want to make sure I understand whether it's preferable to keep using styled-components' `styled()` function versus defining css properties directly in the style object?

Reply:
The advantage being that the arguments to `styled` are a CSS string, so easily moved to a stylesheet if necessary.

Reply:
Thanks!

Reply:
In terms of performance, definitely prefer CSS classes over inline styles on individual elements.

Good:
```jsx
const StyledDiv = styled.div`
  .styledThing {
    color: red;
  }
`

<StyledDiv>
  <p className='styledThing'>Hello</p>
  <p className='styledThing'>World</p>
</StyledDiv>
```

Bad:
```jsx
<StyledDiv>
  <p style={{ color: 'red' }}>Hello</p>
  <p style={{ color: 'red' }}>World</p>
</StyledDiv>
```